backend/fleet_management/tests/test_refuels.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import Group
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from fleet_management.constants import Groups
+from fleet_management.factories import UserFactory
+
+
+class RefuelsApiTestCase(APITestCase):
+    def setUp(self):
+        self.url = reverse("projects")
+        self.user = UserFactory.create(
+            groups=[Group.objects.get(name=Groups.Driver.name)]
+        )
+
+    def test_401_for_unlogged_user(self):
+        res = self.client.get(self.url)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
+
```

Comment:
```suggestion
    def test_can_create_a_refuel(self):
        payload = {
            ...
        }

        self.client.force_login(self.driver)
        res = self.client.post(self.url, data=payload, format="json")

        self.assertEqual(res.status_code, status.HTTP_201_CREATED)
        refuel = Refuel.objects.filter(id=res.data["id"])
        self.assertEqual(refuel.count(), 1)
        self.assertTrue(refuel.currency, "???")
        ...
```

The endpoint should support POST and GET, but there's no test for POST.